### PR TITLE
DOC: Retirement roadmap

### PR DESCRIPTION
**Pull request by pllim**
This might be controversial but decision is necessary, better sooner than later. Open to discussions.

cc @larrybradley @vglaidler

### CODE REVIEW

**Comment by vglaidler** [comment]
The current project planning for the HST ETC continues to depend on pysynphot through CY2020; we currently expect to complete the transition away from pysynphot in late 2021/early 2022. Minimal support as described in the document is sufficient.

This plan has been formulated in consultation with HSTMO. I can ask them to provide a justification to DATB/DMD if desired.

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Roadmap for PySynphot
+
+Some time in 2020, make a last release and archive this package.
```
Comment (pllim) [inline ROADMAP.md:3]: @vglaidler , can the PR be accepted with this change then?
```suggestion
Some time in 2022, make a last release and archive this package.
```

Comment (vglaidler) [reply ROADMAP.md:3]: Yes, looks good to me. Thanks.

**Comment by pllim** [comment]
Given there are no objections and we have sorted out the timeline with ETC, I am merging this. Thanks, everyone!

**Pull request merged**